frontend/javascripts/oxalis/constants.ts

```diff
@@ -337,6 +337,8 @@ const Constants = {
   SCALEBAR_HEIGHT: 22,
   SCALEBAR_OFFSET: 10,
   OBJECT_ID_STRING_LENGTH: 24,
+  REGISTER_SEGMENTS_BB_MAX_VOLUME_VX: 512 * 512 * 512,
+  REGISTER_SEGMENTS_BB_MAX_NO_SEGMENTS: 2000,
```

Comment:
```suggestion
  REGISTER_SEGMENTS_BB_MAX_SEGMENT_COUNT: 2000,
```

"NO" feels like the opposite of yes :smile: